### PR TITLE
Add AddRemarketingAction example

### PR DESCRIPTION
**Pull request by fiboknacky**

### CODE REVIEW

File: examples/Remarketing/AddRemarketingAction.php [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+        // Creates a remarketing action with the specified name.
+        $remarketingAction = new RemarketingAction([
+            'name' => new StringValue(['value' => 'Remarketing action #' . uniqid()])
+        ]);
+
+        // Creates a remarketing action operation.
+        $remarketingActionOperation =
+            new RemarketingActionOperation(['create' => $remarketingAction]);
+
+        // Issues a mutate request to add the remarketing action and print out some information.
```
Comment (PierrickVoulet) [inline examples/Remarketing/AddRemarketingAction.php]: nit: "and print**s**"

File: examples/Remarketing/AddRemarketingAction.php [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+        $googleAdsServiceClient = $googleAdsClient->getGoogleAdsServiceClient();
+        $response =
+            $googleAdsServiceClient->search($customerId, $query, ['pageSize' => self::PAGE_SIZE]);
+
+        // There is only one row because we limited the search using the resource name, which is
+        // unique.
+        /** @var GoogleAdsRow $googleAdsRow */
+        $googleAdsRow = $response->iterateAllElements()->current();
+
+        // Print some attributes of the remarketing action. The ID and tag snippets are generated
```
Comment (PierrickVoulet) [inline examples/Remarketing/AddRemarketingAction.php]: nit: "Print**s**"

File: examples/Remarketing/AddRemarketingAction.php [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+            "Remarketing action has ID %d and name '%s'.%s%s",
+            $googleAdsRow->getRemarketingAction()->getIdUnwrapped(),
+            $googleAdsRow->getRemarketingAction()->getNameUnwrapped(),
+            PHP_EOL,
+            PHP_EOL
+        );
+        print 'It has the following generated tag snippets:' . PHP_EOL;
+        foreach ($googleAdsRow->getRemarketingAction()->getTagSnippets() as $tagSnippet) {
+            /** @var TagSnippet $tagSnippet */
+            printf(
```
Comment (PierrickVoulet) [inline examples/Remarketing/AddRemarketingAction.php]: nit: add tabulation or bullet to delimitate each tag in the list

**Review by PierrickVoulet — APPROVED** [review]
Only nits so approved

File: examples/Remarketing/AddRemarketingAction.php (hunk 3 above)
Comment (fiboknacky) [reply examples/Remarketing/AddRemarketingAction.php]: Thanks for suggestion, but the tags are quite lengthy and hard to read already.
Adding tabulation or bullet will not make it more readable and could confuse some users that those texts are part of the tags.

**Comment by fiboknacky** [comment]
Kokoro runs now fail because of the updated version of the protobuf library.
We're working separately on fixing that.

**Review by sundquist — APPROVED** [review]
LGTM

**Pull request merged**